apps/collection/behaviours/confirm.js

```diff
@@ -32,6 +32,15 @@ module.exports = superclass => class Confirm extends superclass {
     const locals = super.locals(req, res);
     const reason = getReason(req);
     req.sessionModel.set(reason);
+    if (locals.route === 'confirm') {
+      locals.rows = locals.rows.map(row => {
+        if (req.sessionModel.get('nominated-fullname') && row.section === 'Details of the person who attempted to collect your BRP') {
+          row.section = `${req.sessionModel.get('nominated-fullname')}\n\nDetails of the person who attempted to collect your BRP`;
```

Comment:
Are you referring to a string that was created somewhere else i.e. translations?
If you are, what happens if that string changes in the future?

Reply:
Could you explicitly grab that I wonder 🤔

Reply:
👍🏾 Have refactored as suggested

Reply:
ok can't see the change, I'm sure it'll come through

Reply:
change has been made, I'll resolve